app/components/atomic/galoy-tertiary-button/galoy-tertiary-button.tsx

```diff
@@ -0,0 +1,69 @@
+import { makeStyles, Text, useTheme } from "@rneui/themed"
+import React from "react"
+import { Pressable, PressableProps, StyleProp, ViewStyle } from "react-native"
+
+export type GaloyTertiaryButtonProps = {
+  outline?: boolean
+  containerStyle?: StyleProp<ViewStyle>
+  title: string
+} & PressableProps
+
+export const GaloyTertiaryButton = (props: GaloyTertiaryButtonProps) => {
+  const { outline, containerStyle, disabled, ...remainingProps } = props
+  const styles = useStyles(disabled)
+  const { theme } = useTheme()
+  const pressableStyle = ({ pressed }): StyleProp<ViewStyle> => {
+    let dynamicStyle
+    switch (true) {
+      case pressed && outline:
+        dynamicStyle = {
+          borderColor: theme.colors.primary,
+          backgroundColor: theme.colors.primary8,
+          borderWidth: 1.5,
+        }
+        break
+      case pressed && !outline:
+        dynamicStyle = {
+          backgroundColor: theme.colors.primary8,
+        }
+        break
+      case outline:
+        dynamicStyle = {
+          backgroundColor: "transparent",
+          borderColor: disabled ? theme.colors.primary6 : theme.colors.primary,
+          borderWidth: 1.5,
+        }
+        break
+      default:
+        dynamicStyle = {
+          backgroundColor: theme.colors.primary9,
+        }
+    }
+
+    const disabledStyle = disabled ? { opacity: 0.7 } : {}
+
+    const sizingStyle = {
+      paddingHorizontal: 16,
+      paddingVertical: 4,
+      borderRadius: 50,
+    }
+
+    return [sizingStyle, disabledStyle, dynamicStyle, containerStyle]
+  }
+
+  return (
+    <Pressable {...remainingProps} style={pressableStyle} disabled={disabled}>
+      <Text style={styles.buttonTitleStyle}>{props.title}</Text>
+    </Pressable>
+  )
+}
+
+const useStyles = makeStyles((theme, disabled) => ({
+  buttonTitleStyle: {
+    lineHeight: 20,
+    fontSize: 14,
+    fontWeight: "600",
+    color: theme.colors.primary,
+    opacity: disabled === true ? 0.7 : 1,
```

Comment:
no need to equate disabled to true since disabled is a always a boolean. 
```suggestion
    opacity: disabled ? 0.7 : 1,
```

Reply:
`disabled` is actually sometimes an empty object because of how `makeStyles` is implemented, however I will refactor it so that disabled is a property in the object.